cypress/e2e/CMS/$MiniLandingPageTest/index.js

```diff
@@ -103,13 +140,85 @@ And('user selects {string} from Save as dropdown', (title) => {
+And('user selects {string} from Display dropdown', (dropdown) => {
     cy.get("select[name='field_landing_contents[5][subform][field_source_view][0][display_id]']").select(dropdown)
 })
 
 And('user clicks on {string} button to link to a media', (link) => {
     cy.get(`summary[aria-controls^="edit-field-media-link"]`).click({ force: true });
 });
+
+And('Content Heading reads {string}', (title) => {
+    cy.get(`main[id='main-content']:contains('${title}')`)
+})
+
+And('HTML Content reads {string}', (htmlContent) => {
+    cy.get(`p:contains('${htmlContent}')`)
+})
+
+And('feature card row displays the following cards', (dataTable) => {
+    for (const { title, url, description } of dataTable.hashes()) {
+        const siteSection = Cypress.env("test_site_section");
+        const replacedTestSiteSection = url.replace("{TEST_SITE_SECTION}", siteSection);
+        cy.get(`div.feature-card h3:contains('${title}')`).should('be.visible')
+        if (description !== 'N/A') {
+            cy.get(`div.feature-card p:contains('${description}')`).should('be.visible')
+        }
+        cy.get(`div.feature-card a[href="${replacedTestSiteSection}"]`).should('be.visible')
+    }
+})
+
+And('managed list has the following links', (dataTable) => {
+    for (const { title, url, description } of dataTable.hashes()) {
+
+        const replacedTestSiteSection = url.replace("{TEST_SITE_SECTION}", siteSection);
+        cy.get(`div.managed.list h3:contains('${title}')`).should('be.visible')
+        if (description !== 'N/A') {
+            cy.get(`div.managed.list p:contains('${description}')`).should('be.visible')
+        }
+        cy.get(`div.managed.list a[href="${replacedTestSiteSection}"]`).should('be.visible')
+    }
+})
+
+And('borderless card with {string} accent displays the following', (accent, dataTable) => {
+    cy.get(`div.${accent} .borderless-card`).should('be.visible')
+    for (const { title, link } of dataTable.hashes()) {
+        const replacedTestSiteSection = link.replace("{TEST_SITE_SECTION}", siteSection);
+        cy.get(`div.${accent} .borderless-card h2:contains('${title}')`).should('be.visible')
+        cy.get(`div.${accent} .borderless-card a[href="${replacedTestSiteSection}"]`).should('be.visible')
+    }
+})
+
+And('List Item Title is displayed as {string}', (title) => {
+    cy.get(`h2:contains('${title}')`).should('be.visible')
+})
+
+And('the button with text {string} appears with href {string}', (linkText, href) => {
+    const replacedTestSiteSection = href.replace("{TEST_SITE_SECTION}", siteSection);
+    cy.get('a.borderless-button').should('have.text', linkText)
+    cy.get('a.borderless-button').should('have.attr', 'href', replacedTestSiteSection)
+})
+
+And('the Raw HTML Content reads the following {string}', (htmlContent) => {
+    cy.get(`p:contains('${htmlContent}')`)
+})
+
+And('{int} links are displayed under the Dynamic List Title', (num) => {
+    cy.get("a[class='title'] span").should('have.length', num)
```

Comment:
this is not granular enough, e.g. you need the locator to specify its indeed dynamic list's links

such as .dynamic.list a.title